### PR TITLE
Interview dates endpoint

### PR DESCRIPTION
**Pull request by frankyyzy**
#130

### CODE REVIEW

File: src/entities/InductionClass.ts [hunk 1]
```diff
@@ -22,4 +23,10 @@ export class InductionClass {
   // Starting date of the induction cycle for said class
   @Column('date')
   endDate: string;
+
+  @Column({
+    type: 'simple-json',
```
Comment (godwinpang) [inline src/entities/InductionClass.ts:28]: we should use json instead of simple-json here to rely on the native json column type

Comment (godwinpang) [reply src/entities/InductionClass.ts:28]: might have to do more magic with getting types to work nicely idk

File: src/entities/InductionClass.ts [hunk 2]
```diff
@@ -7,7 +7,8 @@ import { Entity, PrimaryColumn, Column } from 'typeorm';
 @Entity()
 export class InductionClass {
   // TODO add validation for what strings are allowed.
-  // e.g. FA20
+
+  // e.g. fa20 (lower case to match string in url)
```
Comment (godwinpang) [inline src/entities/InductionClass.ts:11]: we shouldn't be changing this, existing data is already in upper case

**Review by godwinpang — CHANGES_REQUESTED** [review]

File: src/payloads/InterviewDates.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { IsArray } from 'class-validator';
+
+export class InterviewDatesResponse {
+  @IsArray()
+  interview: Date[];
```
Comment (godwinpang) [inline src/payloads/InterviewDates.ts:5]: We should try to have payloads as strings - Date gets converted into ISO string in json anyways
Better to be explicit here

**Review by godwinpang — CHANGES_REQUESTED** [review]

**Comment by thaigillespie** [comment]
Gonna just merge this now

**Pull request merged**